tests/CI/pilot_newSchema.json

```diff
@@ -101,7 +106,7 @@
 			"pilotRepo": "https://github.com/does_not_matter/Pilot.git",
 			"GenericPilotGroup": "gridpp_pilot",
 			"GenericPilotDN": "VAR_USERDN_GRIDPP",
-			"RemoteLogging": "True",
+			"RemoteLogging": "False",
```

Comment:
@martynia this somehow does not work. Will be tested in other PR.

Reply:
That's strange . It works for me locally and in  #217

Reply:
OK, now I see. You changed the schema and the test (obviously) failed. Now after "fixing" the test everything passes. A question remains why the original setting (with the remote logger ON) was not acceptable. This poses a potential problem when merging this and rebasing #217, where the logging is on by default.